017-FrontDoor/Coach/Solution04.md

```diff
@@ -0,0 +1,12 @@
+### Notes for Challenge 4 - Cache Static Assets
+
+ [< Previous Challenge [3]](./Solution03.md) - **[Home](./README.md)**
+
+
+This article [https://docs.microsoft.com/en-us/azure/frontdoor/front-door-tutorial-rules-engine](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-tutorial-rules-engine)
```

Comment:
What about "this article" ?? Give us a little more context.

Reply:
yea it sounded better in my head before I started to type